pkgs/by-name/ol/ollama/package.nix

```diff
@@ -156,8 +165,7 @@ goBuild ((lib.optionalAttrs enableRocm {
   ] ++ lib.optionals (enableRocm || enableCuda) [
     makeWrapper
     autoAddDriverRunpath
-  ] ++ lib.optionals stdenv.isDarwin
-    metalFrameworks;
+  ];
```

Comment:
From what I remember, the metal frameworks needed to be in both `nativeBuildInputs` and `buildInputs` for `ollama` to build right on darwin/macos. If you leave this in, we'll need to verify that ollama works right on darwin.